src/plugins/sync-plugin.js

```diff
@@ -99,7 +99,7 @@ export const ySyncPlugin = (yXmlFragment, { colors = defaultColors, colorMapping
       },
       apply: (tr, pluginState) => {
         const change = tr.getMeta(ySyncPluginKey)
-        if (change !== undefined) {
+        if (change !== undefined && typeof change === 'object') {
```

Comment:
Please remove this change. I don't think it makes sense to introduce the value `true` when there already is a `isChangeOrigin` property.